index.html

```diff
@@ -67,122 +85,125 @@ <h1>Hey there. I’m Mamun</h1>
                 </li>
             </ul>
         </div>
+        <div id="down-arrow">
+            <img src="assets/img/Action.png">
+        </div>
     </section>
-    <div id="down-arrow">
-        <img src="assets/img/Action.png">
-    </div>
 
     <section id="recent-work" class="c-posts">
         <h2 class="section-title">My Recent Works</h2>
-        <article class="c-posts__item card-wrapper">
-            <div class="card-image-wrapper"></div>
-            <div class="card-content-wrapper">
-                <h2 class="work-card-title">Multi-Post Stories Gain + Glorys</h2>
-                <div class="tech-tags">
-                    <ul>
-                        <li><a href="#">Ruby on rails</a></li>
-                        <li><a href="#">css</a></li>
-                        <li><a href="#">JavaScript</a></li>
-                        <li><a href="#">html</a></li>
-                    </ul>
+        <div class="items">
```

Comment:
- [ ] This `items` container that holds `recent works` and `about me` sections doesn't have enough margin on both sides above `768px`. kindly consider giving this container enough margin on both sides so that it looks much better because for the beautiful and aesthetic design, we need to stick to the figma design.
your design
![asd](https://user-images.githubusercontent.com/96683843/214650820-403496c9-3a59-4f48-b4d7-9edda26895c4.jpg)
![dfgh](https://user-images.githubusercontent.com/96683843/214651561-3978a3a4-5eac-4074-9e70-74708d734475.jpg)


figma design
![try](https://user-images.githubusercontent.com/96683843/214650921-19fe28ec-9bd3-404c-a86c-dfa58a6784fb.jpg)
![fdsa](https://user-images.githubusercontent.com/96683843/214651364-01eb35f3-3546-4137-9165-fec5334398a9.jpg)